include/psa/crypto_values.h

```diff
@@ -932,6 +939,33 @@
 #define PSA_MAC_TRUNCATED_LENGTH(mac_alg)                               \
     (((mac_alg) & PSA_ALG_MAC_TRUNCATION_MASK) >> PSA_MAC_TRUNCATION_OFFSET)
 
+/** Macro to build a MAC minimum-MAC-length wildcard algorithm.
+ *
+ * A minimum-MAC-length MAC wildcard algorithm permits all MAC algorithms
```

Comment:
I'd be more comfortable with something like:
```suggestion
 * A key with a minimum-MAC-length MAC wildcard algorithm as permitted algorithm policy can be used with all MAC algorithms
```

Reply:
The current wording was already approved by both @athoelke and @gilles-peskine-arm. I'd rather not have this drag out for another couple of weeks in a verbiage discussion.

Reply:
ok, I am not very comfortable with it but if you got already approval for it, let's stick to that.